dev/managedservicesplatform/internal/resource/alertpolicy/alertpolicy.go

```diff
@@ -105,12 +105,17 @@ type ResponseCodeMetric struct {
 	Duration     *string
 }
 
-type SeverityLevel string
+// MQL for alerting when a MQL query exceeds a threshold
+type MQL struct {
+	Query    string
+	Duration *string
```

Comment:
Let's mention the default
```suggestion
	// Duration is how often the query is ran, and the default is "60s".
	Duration *string
```

What does "Duration" mean here exactly?

Reply:
"The amount of time that a time series must violate the threshold to be considered failing"

Reply:
Got it, could you add that to the docstring instead?